cmake/packaging/linux.cmake

```
@@ -79,7 +79,8 @@ if(${SUNSHINE_TRAY} STREQUAL 1)
 
     set(CPACK_DEBIAN_PACKAGE_DEPENDS "\
                     ${CPACK_DEBIAN_PACKAGE_DEPENDS}, \
-                    libappindicator3-1")
+                    libayatana-appindicator3-1, \
+                    libnotify4")
```

This will be sufficient. `libnotify4` is a dependency of `libc6`, which is a dependency of `libayatana-appindicator3-1`.

```suggestion
                    libayatana-appindicator3-1)
```

---

I think you've got the dependency direction flipped. libnotify4 depends on libc6, not the other way around.

---

Thanks for reviewing! One clarification first though, I agree that `libc6` is a dependency of `libayatana-appindicator3-1`, but it seems to me that `libc` is also a dependency of `libnotify4` (not the other way around).

Using the docker image, if I install the Sunshine deb and `libayatana-appindicator3-1` I am still lacking `libnotify4.so.4`. It seems to me that `libnotify4` is necessary, do you agree?

---

@cgutman Looks like you just beat me to it :)

---

Okay, my mistake.